openquake/ghm/create_map_from_curves.py

```diff
@@ -104,7 +104,7 @@ def main(argv):
     p.arg(name='prefix', help='Prefix for selecting files')
     p.arg(name='fname_out', help='Name output csv file')
     p.arg(name='path_out', help='Path to the output folder')
-    p.arg(name='iml_str', help='String describing the IML')
+    p.arg(name='imt_str', help='String describing the IMT')
```

Comment:
You're right. Shouldn't we need to also change the name of the variable accordingly?

Reply:
The name is correct here: https://github.com/GEMScienceTools/oq-mbtk/blob/master/openquake/ghm/bin/create_map_from_curves.sh#L17